torch/_inductor/cpu_vec_isa.py

```diff
@@ -306,9 +320,21 @@ def _check_and_append_supported_isa(
 
     return supported_isa
 
+def _is_arm_neoverse_v1() -> bool:
```

Comment:
It would be nice to memoize the results, as they will not change from one call to another, but opening a file is technically an expensive procedure for a function that could be called 1000 times
```suggestion
@functools.memoize()
def _is_arm_neoverse_v1() -> bool:
```

Reply:
ack. done.